app/templates/courses/enrollments-view.jade

```diff
@@ -21,6 +21,13 @@ block content
           h5(data-i18n='teacher.what_is_a_teacher_account')
           p(data-i18n='teacher.teacher_account_explanation')
 
+  else if me.get('clientPermissions') && !me.get('clientPermissions').manageLicensesViaUI
```

Comment:
Can collapse to one `.get()` call: `!(me.get('clientPermissions') || {}).manageLicensesViaUI`

Reply:
Or, maybe we should add a `me.canManageLicensesViaUI()` method, since we do this a lot.